tests/utilities/test_flow_decorator.py

```diff
@@ -135,18 +137,7 @@ def test_decorators_with_flow_decorator(
     # THEN the span are returned bottom to top
     assert read_from_opentelemetry_span(span=spans[1], key=HUMANLOOP_FILE_KEY)["tool"]
     assert read_from_opentelemetry_span(span=spans[2], key=HUMANLOOP_FILE_KEY)["prompt"]
-    # assert read_from_opentelemetry_span(span=spans[3], key=HL_FILE_OT_KEY)["flow"]
```

Comment:
No longer relying on TRACE_FLOW_CONTEXT here, using the ancestor to determine that logs are uploaded correctly.